communities/microgalaxy/lab/sections/4_tools.yml

```diff
@@ -1,5 +1,7 @@
 id: tools
 title: Community curated tools
+exclude_from:
```

Comment:
Why do you exclude Galaxy Australia there?

Reply:
There are too many tools in this section, and they are not described in any detail. As I suggested to Paul [here](https://github.com/galaxyproject/galaxy_codex/issues/236#issuecomment-2479820907) this should really be implemented as a tool panel view, as seen on https://proteomics.usegalaxy.org.au/.

Reply:
See [Add microgalaxy tool panel view](https://github.com/galaxyproject/galaxy_codex/pull/305/commits/e7c9a021551bee51915af5f6946385cabd110785) - This can be selected for microgalaxy.usegalaxy.* using the `default_panel_view_by_host`  `galaxy.yml` config var.